### PR TITLE
Small fixes and enhancements

### PR DESCRIPTION
**Pull request by ghost**
This change is [Reviewable](https://reviewable.io/reviews/lkl/linux/219)

### CODE REVIEW

File: tools/lkl/tests/boot.c [hunk 1]
```diff
@@ -112,6 +112,60 @@ int test_getpid(char *str, int len)
...
+int test_syscall_latency(char *str, int len)
+{
+	struct timespec start, stop;
+	static const int count = 1000;
+	int i;
+	long delta, min = 1000000000, max = -1, avg;
+	int tmp;
+
+	for (i = 0; i < count; i++) {
+		clock_gettime(CLOCK_MONOTONIC, &start);
```
Comment (thehajime) [inline tools/lkl/tests/boot.c]: Is it possible to use lkl_host_ops to obtain current a wall clock value ? It'll be host independent then.

Comment (Rondom) [inline tools/lkl/tests/boot.c]: `time_ns` provides ns since 1970 (under Win32 only with 100ns resolution)

File: tools/lkl/tests/boot.c [hunk 2]
```diff
@@ -112,6 +112,60 @@ int test_getpid(char *str, int len)
...
+		delta = 1e9*(stop.tv_sec - start.tv_sec) +
+			(stop.tv_nsec - start.tv_nsec);
+		if (min > delta)
+			min = delta;
+		if (max < delta)
+			max = delta;
+		if (i == 0)
+			avg = delta;
+		else
+			avg = (avg + delta) / 2;
```
Comment (liuyuan10) [inline tools/lkl/tests/boot.c]: why don't calculate the accurate average?

**Comment by opurdila** [comment]
Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---

*[tools/lkl/tests/boot.c, line 125 \[r1\]](https://reviewable.io:443/reviews/lkl/linux/219#-KQCreNs5olYIzBhjA8n-r1-125:-KQUplR8Ozk7Q-yx-poY:b-896fix) ([raw file](https://github.com/lkl/linux/blob/c252d53bfabd8bf2037195bc06526e3ffb79c218/tools/lkl/tests/boot.c#L125)):*
<details><summary><i>Previously, Rondom wrote…</i></summary>
> `time_ns` provides ns since 1970 (under Win32 only with 100ns resolution)
</details>
Done.

---

*[tools/lkl/tests/boot.c, line 159 \[r1\]](https://reviewable.io:443/reviews/lkl/linux/219#-KQCreNs5olYIzBhjA8n-r1-159:-KQUplz2iTRGHKww8D5_:b-896fix) ([raw file](https://github.com/lkl/linux/blob/c252d53bfabd8bf2037195bc06526e3ffb79c218/tools/lkl/tests/boot.c#L159)):*
<details><summary><i>Previously, liuyuan10 (Yuan Liu) wrote…</i></summary>
> why don't calculate the accurate average?
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/lkl/linux/219)*

**Comment by thehajime** [comment]
:lgtm:

---

Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/lkl/linux/219#-:-KQUrkdS7wphpheGbj0V:bnfp4nl)*

**Comment by liuyuan10** [comment]
lgtm

**Pull request merged**